src/main/java/com/booking/app/controller/HotelController.java

```diff
@@ -0,0 +1,21 @@
+package com.booking.app.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HotelController extends BaseController {
+    public static final String API_NAME = "hotels";
+
+    @GetMapping(HotelController.API_NAME + "/{id}")
+    public ResponseEntity<String> getHotel(@PathVariable long id) {
+        return ResponseEntity.ok("Hotel Y");
+    }
+
+    @GetMapping("/hotels")
```

Comment:
Please be more consistent regarding the API URI. The above method uses the HotelController.API_NAME but this one does not. Applies everywhere